server/server.go

```diff
@@ -96,11 +96,13 @@ func run(ctx context.Context, addr, tlsCertFile, tlsKeyFile string, trust signed
 	}
 	tlsLsnr := tls.NewListener(lsnr, tlsConfig)
 
-	var ac auth.AccessController = nil
-	//ac, err := auth.GetAccessController("token", map[string]interface{}{})
-	//if err != nil {
-	//	return err
-	//}
+	if authMethod == "" {
+		authMethod = "silly"
```

Comment:
@diogomonica Silly auth is just for testing functionality of the authorization stack. Adding a test here and defaulting to is not the right thing thing to do.